src/test/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/integration/admin/AdminUpdateSessionTemplateTest.kt

```diff
@@ -162,7 +162,7 @@ class AdminUpdateSessionTemplateTest : IntegrationTestBase() {
       sessionTemplate = sessionTemplateDefault,
       slotDate = LocalDate.now(),
       visitStart = dto.sessionTimeSlot!!.startTime,
-      visitEnd = dto.sessionTimeSlot!!.endTime,
+      visitEnd = dto.sessionTimeSlot.endTime,
```

Comment:
updated based on Intellij recommendation as sessionSlot!! is done on the previous line.